src/injected/tab-stops-requirement-evaluator.ts

```diff
@@ -31,12 +31,12 @@ export interface TabStopsRequirementEvaluator {
 
 export class DefaultTabStopsRequirementEvaluator implements TabStopsRequirementEvaluator {
     private readonly keyboardNavigationDescription: (string) => string = selector =>
-        `Element ${selector} was expected, but not reached in tab order`;
+        `[Automatically detected, needs review] Unreachable element: ${selector}.`;
```

Comment:
Same as previous comment; since "[Automatically detected, needs review]" is repeated, should we store that in a common variable? This is also minor since there are currently only 3 such requirements